src/components/Cards/Cards.tsx

```diff
@@ -39,6 +35,7 @@ const Cards = () => {
 				<Card
 					suit={suit}
 					name={name}
+					isHide={index in removedCards}
```

Comment:
Use `Object.prototype.hasOwnProperty.call`. Operator `in` considered as unsafe in terms of inheritance